src/app/[locale]/seller/registration/page.tsx

```diff
@@ -388,6 +401,34 @@ const SellerRegistrationForm = () => {
           <ToggleCollapse
             header={t('SCREEN.BUY_FROM_SELLER.SELLER_CONTACT_DETAILS_LABEL')}
             open={false}>
+                <div className="mb-4">
+                <label>
+                  {t('SCREEN.SELLER_REGISTRATION.EMAIL_LABEL')}
+                </label>
+                <p className="text-gray-400 text-sm mt-1">
+                </p>
+                <Input
+                  name="email"
+                  type="text"
+                  value={formData.email}
+                  onChange={handleChange}
+                />
+              </div>
+              <div className="mb-4">
+                {/* Phone input */}
+                <label>
+                  {t('SCREEN.SELLER_REGISTRATION.PHONE_NUMBER_LABEL')}
+                </label>
+                <Input
+                  name="phone_number"
+                  type="text"
+                  value={formData.phone_number}
+                  onChange={handleChange}
+                />
```

Comment:
Based on my understanding from yesterday's standup, I believe Phil wanted the editable fields to be positioned as the last elements on the screen, with the public note placed just above the Save button—unless there were any changes to that plan after the meeting.

Here is what I foresee ->

![image](https://github.com/user-attachments/assets/10553568-786d-4412-8d1b-4e21ab88a734)

Reply:
Oh ok, if that's the case let's move it where it should be